problemsets-mat223/modules/module11-exercises.tex

```diff
@@ -12,6 +12,28 @@
 			\item $M_{4}=\mat{1&-2&0&-1\\3&5&-1&0\\2&3&-2&0\\0&0&0&1}$.
 		\end{enumerate}
 
+			% Q1 Solutions
+		\begin{solution}
+		    \begin{enumerate}
+    		    \item $M_1 = \mat{1&2&1\\3&1&-2\\8&6&-2}$ \newline
+    		    $\Null(M_{1})=\Set*{\vec{x} \in \R^3: M_1 \vec{x}=\vec 0}$ \newline
```

Comment:
No manual newlines. Fix throughout and remove all `\newline` and `\\` that are used to force a newline.